ui/src/app/tasks-jobs/tasks/launch/builder/builder.component.html

```diff
@@ -140,10 +142,10 @@
                   </clr-tooltip>
                 </div>
 
-                <div class="form-control form-control-label">Version</div>
+                <div class="form-control form-control-label">{{ 'tasks.launch.builder.version' | translate }}</div>
               </div>
               <div class="line-body">
-                <div class="form-control form-control-label">Properties</div>
+                <div class="form-control form-control-label">{{ 'tasks.launch.builder.properties' | translate }}</div>
```

Comment:
Texts like `Version` and `Properties` are visible on other pages.
Can we move these to `commons` translations ?